benchmarks/test_bocquet20_hmf.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import pyccl as ccl
+
+
+def test_bocquet20_mf():
+    d = np.load("./benchmarks/data/mf_bocquet20mf.py.npz")
```

Comment:
Well, just in case the package ever stops being maintained and we've reimplemented it. This also follows the same pattern we use for virtually all other benchmarks.